sips/minutes/_posts/2016-08-20-sip-20th-september-minutes.md

```diff
@@ -0,0 +1,239 @@
+spore class definition to avoid the automatic conversion to a SAM type. This has
+no performance hit on the eventual representation of a spore into JVM classes,
+since JVM SAM types cannot be used.
+
+Heather joins the conversation. Jorge explains the current discussion and Martin
+asks her what's her opinion on not including spores into the compiler. She
+says that could be left out and she mentions that spores can only provide a
+shallow analysis, a "skin-deep" check of what's captured.  Pickling provided a
+transitive way of checking the spores contract, but the proposal doesn't take it
+into account. Everyone agrees that the proposal should be more general and only
+focus on ways to control the captured environment. Sébastien proposes a
+typeclass-based way of overcoming the current limitations of spores, but it's
+yet to be seen its feasibility in practice. Martin suggests that as the
+Committee should analyze spores as it is right now. Heather emphasizes that the
+current design is not very useful, so she asks whether we should keep it as such
+or try to improve it.
+
+Martin encourages to improve the spores proposal, but suggests to close the
+proposal because it's not yet ready. Heather proposes to give the proposal
+another iteration to update and improve it.
+
+**Outcome**: The spores proposal will have another iteration in two months
+(November). By then, Heather is asked to update the proposal and try to find out
+a way to provide more thorough checks (spores transitivity).
+
+### Discussion of SIP-26: Unsigned Integer Data Types
+
+Sébastien introduces his updates on the proposal. He's tried hard to remove the
+performance regression in his implementation, but he hasn't found a way. He
+describes that the implementation is still 6% slower because of performance hits
+in the hashCode and equals implementation of a case class. To work around this
+issue, two things are required: 
+
+  * Changing the underlying implementation of byte and short ints.
+  * Making the super class of unsigned integer extend *java.lang.Number*
+
+The second option is not feasible because some numbers are AnyVals, and they can
+only extend objects. Working around this in the backend is, in Sébastien's
+opinion, not an exciting adventure to embark on: a lot of patches and quirky
+fixes are required in the compiler. Sébastien, recognizing his unability to fix
+the issue, recommends to reject the proposal.
+
+Josh needs to leave. Eugene wonders if these problems are only JVM-specific.
+Sébastien replies that both yes and no, and he confirms that unsigned integers
+will be implemented in Scalajs alone, so the implementation won't be
+platform-independent. Eugene is interested in knowing if there will be any code
+duplication in the implementation, and Sébastien doesn't think so, since Scala
+Native implements unsigned integer in a different way.
+
+**Outcome**: The proposal is rejected by majority (Heather is the only Committee
+member voting against, 5 to 1).
```

Comment:
Ah, I thought I said "not in favor". It was either a connectivity issue or a brainslip.